src/builder.rs

```diff
@@ -597,6 +602,6 @@ const fn get_texture_format_size(texture_format: wgpu::TextureFormat) -> f32 {
         Astc { block: B12x10, channel: _ } => 7.5, // 12.0 * 10.0 / 16.0
 
         // 12x12 blocks, 16 bytes per block
-        Astc { block: B12x12, channel: _ } => 9.0, // 12.0 * 12.0 / 16.0
+        Astc { block: B12x12, channel: _ } => 9.0,
```

Comment:
Please revert this change. The comment is there so that we can replace the single arbitrary float with an equation that matches the comments. Const float arithmetic is not currently supported in Rust.